app/controllers/idv/review_controller.rb

```diff
@@ -92,7 +91,7 @@ def flash_message_content
     end
 
     def idv_profile_complete?
-      idv_session.profile_confirmation == true
+      idv_session.resolution_successful == true
```

Comment:
Why is the definitely of profile_complete changing here?

Reply:
I did this because I found a spot where the verify info controller explicitly set this. This was before I saw it _also_ implemented `save_legacy_state`. It also turned out to need a little massaging to get in-person proofing to work so I will probably wind this back.